tests/decorator-free.test.ts

```diff
@@ -433,15 +434,8 @@ describe('decorator-free-tests', () => {
     expect(steps[0].childWorkflowID).toBeNull();
   });
 
-  test('wf-inst-step-reg-swf', async () => {
-    const handle = await DBOS.startWorkflowFunction(
-      {
-        queueName: queue.name,
-        instance: inst,
-      },
-      inst.wfRegStep,
-      10,
-    );
+  test('wf-inst-step-reg-startWorkflow', async () => {
```

Comment:
Can we add some tests for the new startWorkflow overload that handles free functions? All those tests are class/instance methods.

Reply:
https://github.com/dbos-inc/dbos-transact-ts/blob/475679cd446a61f6cef4d1c34cda8d8f9a010c25/tests/decorator-free.test.ts#L279 is a free function test

Reply:
https://github.com/dbos-inc/dbos-transact-ts/blob/475679cd446a61f6cef4d1c34cda8d8f9a010c25/tests/decorator-free.test.ts#L219

Reply:
Oh sorry I missed this.